nilearn/interfaces/fmriprep/tests/utils.py

```diff
@@ -51,53 +86,91 @@ def get_testdata_path(non_steady_state=True):
 def create_tmp_filepath(
     base_path,
     image_type="regular",
-    suffix="sub-test01_task-test",
+    bids_fields=None,
     copy_confounds=False,
     copy_json=False,
     old_derivative_suffix=False
 ):
+    entities_to_include = [
```

Comment:
`entities_to_include` should be a kw argument ?

Reply:
In this case this function will only ever create derivatives filenames so I do not think that it makes sense to add more flexiblity to it.

if they want more flexiblity then it is better to:

```
from nilearn._utils.data_gen import _create_bids_filename
```

as  `_create_bids_filename` has this `entities_to_include` as kwarg